### PR TITLE
Adding help for redirection

### PR DESCRIPTION
**Pull request by faridtsl**
Adding help for output redirection #9628 .

### CODE REVIEW

File: libr/core/cmd.c [hunk 1]
```diff
@@ -1930,6 +1930,11 @@ static int r_core_cmd_subst_i(RCore *core, char *cmd, char *colon) {
 			return r_cmd_call (core->rcmd, cmd);
 		}
 		break;
+	case '?':
+		if ( cmd[1] && cmd[1] == '>' ) {
```
Comment (radare) [inline libr/core/cmd.c]: remove space after ( and before ). also the cmd[1] is implicit in cmd[1]=='>'

File: libr/core/cmd.c [hunk 2]
```diff
@@ -2125,6 +2130,10 @@ static int r_core_cmd_subst_i(RCore *core, char *cmd, char *colon) {
 	ptr = (char *)r_str_firstbut (cmd, '>', "\"");
 	// TODO honor `
 	if (ptr) {
+		if (ptr[1] && ptr[1] == '?') {
```
Comment (radare) [inline libr/core/cmd.c]: same here, ptr[1]=='?' makes ptr[1] implicit. you should check ptr[0] instead

File: libr/core/cmd_help.c [hunk 3]
```diff
@@ -166,6 +167,15 @@ static const char *help_msg_question_V[] = {
 	NULL
 };
 
+static const char *help_msg_greater_sign[] = {
+    "Usage:", "[cmd]>[file]", "Redirects the console output of 'cmd' to 'file'",
```
Comment (radare) [inline libr/core/cmd_help.c]: we use tabs

Comment (faridtsl) [reply libr/core/cmd_help.c]: Sorry copy/pasta

**Pull request merged**